### PR TITLE
Bug 558835 LIC Execute feature must have it's own branding plug-in

### PR DESCRIPTION
**Pull request by eparovyshnaya**
- invite new `org.eclipse.passage.lic.execute.branding` plug-in
- connect it to `lic.execute.feature` as branding plugin

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.execute.branding/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.eclipse.passage.lic.execute.branding
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: org.eclipse.passage.execute.branding
+Bundle-Version: 0.1.0.qualifier
+Bundle-Name: %pluginName
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.execute.branding/META-INF/MANIFEST.MF]: This is the legacy style, please use OSGI-INF/l10n

**Comment by ruspl-afed** [comment]
Nope, LOL

File: features/org.eclipse.passage.lic.execute.feature/feature.xml [hunk 2]
```diff
@@ -16,6 +16,7 @@
       label="%featureName"
       version="0.8.0.qualifier"
       provider-name="%providerName"
+      plugin="org.eclipse.passage.execute.branding"
```
Comment (ruspl-afed) [inline features/org.eclipse.passage.lic.execute.feature/feature.xml]: Please update year in the header

**Comment by eparovyshnaya** [comment]
Resolves [558835](https://bugs.eclipse.org/bugs/show_bug.cgi?id=558835)

**Pull request merged**